datasetinsights/evaluation_metrics/__init__.py

```diff
@@ -12,8 +17,12 @@
     IoU,
```

Comment:
__all__ should be a list of strings instead of list of class names. 

https://docs.python.org/3/tutorial/modules.html#importing-from-a-package